typescript/helloworld/package.json

```diff
@@ -59,5 +59,12 @@
     "test": "hardhat test ./src/test/**/*.test.ts",
     "sync": "ts-node scripts/sync-with-template-repo.ts"
   },
-  "types": "dist/src/index.d.ts"
+  "types": "dist/src/index.d.ts",
+  "resolutions": {
```

Comment:
We need these here too (in addition to root) so when this gets copied to template repo, we get resolutions there too